src/app/error.tsx

```
@@ -0,0 +1,62 @@
+'use client'
+
+import { useMounted } from '@hooks/useMounted'
+import Image from 'next/image'
+import { useTheme } from 'next-themes'
+
+export default function Error({ reset }: { reset?: () => void }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)